package/remarkable_math_graph/package

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(remarkable_math_graph)
+pkgdesc="An example template for the reMarkable tablet"
+url=https://github.com/PeterGrace/remarkable_math_graph
+pkgver=0.0.2-1
+timestamp=2021-01-14T02:03Z
+section=utils
+maintainer="Peter Grace <pete.grace@gmail.com>"
+license=MIT
+depends=(templatectl)
+
+image=qt:v1.2.2
+source=(https://github.com/PeterGrace/remarkable_math_graph/archive/v0.0.2.zip)
+sha256sums=(f782efc048af932bd34a2390f757aecb9b5b422980ed516b99431ac0f8891926)
+
+build() {
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update -y
+    apt-get install -y \
+        --no-install-recommends \
+        -o Dpkg::Options::="--force-confdef" \
+        imagemagick librsvg2-bin
+	convert -scale 26.464% remarkable_math_graph_qcadexport.svg \
+        remarkable_math_graph.svg
+    convert remarkable_math_graph.svg remarkable_math_graph.png
+}
+
+package() {
+    install -D -m 755 -t "$pkgdir"/usr/share/remarkable/templates \
```

Comment:
Perhaps these should be stored in /opt and templatectl handles copying them in?

Reply:
Agreed. Symlinks are even better.

Reply:
If we're going to be copying it to the right spot eventually, why would we first copy it to /opt and then obfuscate its installation by doing it inside of a program that the user is unaware of the intermediate steps for?

templatectl is meant to interact with templates.json only.  It's not meant as an installer for templates, but as a way to properly format xochitl's datafile that it uses to load templates.  I'd prefer to keep its use case succinct.

Reply:
Having them deploy to /usr requires them to be reinstalled on device upgrade. If we have a location in /opt, we could easily iterate through the items and reinstall them as part of the reenable script.

Reply:
OK -- then I'm in agreement with @danshick  then -- install into /opt/templates and the the enable scripting will ensure that a symbolic links exist?

Reply:
`/opt/share/templates` or similar would probably be better. The reeneable script could then loop through any package that has files in those locations and rerun the configure call to add the symlink and get `templatectl` to add them to the json file.
@matteodelabre thoughts?

Reply:
@Eeems Maybe another use-case for a bind mount that we can restore in the reenable script?

Reply:
@matteodelabre I was thinking that as well.